chanacceptor/zeroconfacceptor.go

```diff
@@ -0,0 +1,53 @@
+package chanacceptor
+
+// ZeroConfAcceptor wraps a regular ChainedAcceptor. If no acceptors are in the
+// ChainedAcceptor, then Accept will reject all channel open requests. This
```

Comment:
> If no acceptors are in the ChainedAcceptor, then Accept will reject all channel open requests
Shouldn't this instead only deny the funding requests that want a _zero_ conf channel? 

Or is this intended to be also compatible w/ some other implementations that may not actually set the channel type bit? So then this would effectively force any systems/users that allow zero conf to accept/deny _each_ funding requests if the bit is active?

Reply:
Tbh, I think that's ok, they really _should_ be setting it imo. And AFAIK, in cases where ppl want to use `lnd` with impls that don't set the bit, they sort of already know it, so they'd have the acceptor hooked up.

Reply:
Changed to account for this